rearrange/procthor_rearrange/tasks.py

```diff
@@ -0,0 +1,1256 @@
+"""Include the Task and TaskSampler to train on a single unshuffle instance."""
```

Comment:
Doing a diff between this and `rearrange/tasks.py` suggests to me that we might (and probably should) use the same task for both projects and just have a few switches here and there that tell us what environment we should use and how we should load data.

Reply:
I slightly modified the original task and subclassed for `ProcTHOR`. Not exactly the same task, but almost...